src/hooks/util/usePolling.ts

```diff
@@ -10,7 +10,7 @@ export const usePolling = (callback: Function, delay?: number | null, timeout?:
    const [hasTimedOut, _, restart] = useTimeout(timeout);
 
    /* hasTimedOut returns true if the timeout has been reached */
-   const pollingPeriod = !hasTimedOut() ? delay : null;
+   const pollingPeriod = timeout === undefined || !hasTimedOut() ? delay : null;
```

Comment:
yea. well false then true
![image](https://github.com/user-attachments/assets/5264dd4e-9cd8-4fac-9707-e8e5ba3f2878)

Reply:
ah yeah it is because it in this case sets the delay value to 0 which means process function at next tick. I would just update this line to also check for timeout being null so instead of `timeout === undefined` I'd use `timeout == null`

Reply:
Why check for null? The  type of timeout is undefined or number.

Reply:
good point. I forgot that only delay is nullable. ignore